{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!jupyter nbconvert --to markdown geo_parsing.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from Bio import Geo\n",
    "import gzip"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = open('GSE68849.txt')\n",
    "records = Geo.parse(handle)\n",
    "for record in records:\n",
    "    print(record)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DATABASE \n",
      "!Database_name = Gene Expression Omnibus (GEO)\n",
      "!Database_institute = NCBI NLM NIH\n",
      "!Database_web_link = http://www.ncbi.nlm.nih.gov/geo\n",
      "!Database_email = geo@ncbi.nlm.nih.gov\n",
      "SERIES \n",
      "!Series_title = Transcriptional response of Sacchromyces cerevisiae to change in oxygen provision\n",
      "!Series_geo_accession = GSE22832\n",
      "!Series_status = Public on Apr 01 2011\n",
      "!Series_submission_date = Jul 08 2010\n",
      "!Series_last_update_date = Feb 21 2017\n",
      "!Series_pubmed_id = 21348598\n",
      "!Series_summary = In industrial fermentations of Saccharomyces cerevisiae, transient changes in oxygen concentration commonly occur and it is important to understand the behaviour of cells during these changes. Saccharomyces cerevisiae CEN.PK113-1A was grown in glucose-limited chemostat culture with 1.0% and 20.9% O2 in the inlet gas (D= 0.10 /h, pH5, 30C). After steady state was achieved, oxygen was replaced with nitrogen and cultures were followed until new steady state was achieved. The overall responses to anaerobic conditions of cells initially in different conditions were very similar. Independent of initial culture conditions, transient downregulation of genes related to growth and cell proliferation, mitochondrial translation and protein import, and sulphate assimilation was seen. In addition, transient or permanent upregulation of genes related to protein degradation, and phosphate and amino acid uptake was observed in all cultures. However, only in the initially oxygen-limited cultures was a transient upregulation of genes related to fatty acid oxidation, peroxisomal biogenesis, oxidative phosphorylation, TCA cycle, response to oxidative stress, and pentose phosphate pathway observed. Furthermore, from the initially oxygen-limited conditions, a rapid response around the metabolites of upper glycolysis and the pentose phosphate pathway was seen, while from the initially fully aerobic conditions, a slower response around the pathways for utilisation of respiratory carbon sources was observed.\n",
      "!Series_overall_design = Time series analysis starting from two (1% and 20.9%) levels of oxygen provision. Seven timepoints from both time series and two biological replicates from each timepoint were analysed.\n",
      "!Series_type = Expression profiling by array\n",
      "!Series_sample_id = GSM564449\n",
      "!Series_sample_id = GSM564450\n",
      "!Series_sample_id = GSM564451\n",
      "!Series_sample_id = GSM564452\n",
      "!Series_sample_id = GSM564453\n",
      "!Series_sample_id = GSM564454\n",
      "!Series_sample_id = GSM564455\n",
      "!Series_sample_id = GSM564456\n",
      "!Series_sample_id = GSM564457\n",
      "!Series_sample_id = GSM564458\n",
      "!Series_sample_id = GSM564459\n",
      "!Series_sample_id = GSM564460\n",
      "!Series_sample_id = GSM564461\n",
      "!Series_sample_id = GSM564462\n",
      "!Series_sample_id = GSM564463\n",
      "!Series_sample_id = GSM564464\n",
      "!Series_sample_id = GSM564465\n",
      "!Series_sample_id = GSM564466\n",
      "!Series_sample_id = GSM564467\n",
      "!Series_sample_id = GSM564468\n",
      "!Series_sample_id = GSM564469\n",
      "!Series_sample_id = GSM564470\n",
      "!Series_sample_id = GSM564471\n",
      "!Series_sample_id = GSM564472\n",
      "!Series_sample_id = GSM564473\n",
      "!Series_sample_id = GSM564474\n",
      "!Series_sample_id = GSM564475\n",
      "!Series_sample_id = GSM564476\n",
      "!Series_contact_name = Eija,,Rintala\n",
      "!Series_contact_email = eija.rintala@vtt.fi\n",
      "!Series_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Series_contact_address = P.O.BOX 1000\n",
      "!Series_contact_city = VTT\n",
      "!Series_contact_zip/postal_code = 02044\n",
      "!Series_contact_country = Finland\n",
      "!Series_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE22nnn/GSE22832/suppl/GSE22832_RAW.tar\n",
      "!Series_platform_id = GPL2529\n",
      "!Series_platform_taxid = 4896\n",
      "!Series_platform_taxid = 4932\n",
      "!Series_sample_taxid = 4932\n",
      "!Series_relation = BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA127885\n",
      "PLATFORM \n",
      "!Platform_title = [Yeast_2] Affymetrix Yeast Genome 2.0 Array\n",
      "!Platform_geo_accession = GPL2529\n",
      "!Platform_status = Public on Jun 13 2005\n",
      "!Platform_submission_date = Jun 09 2005\n",
      "!Platform_last_update_date = Feb 21 2017\n",
      "!Platform_technology = in situ oligonucleotide\n",
      "!Platform_distribution = commercial\n",
      "!Platform_organism = Schizosaccharomyces pombe\n",
      "!Platform_organism = Saccharomyces cerevisiae\n",
      "!Platform_taxid = 4896\n",
      "!Platform_taxid = 4932\n",
      "!Platform_manufacturer = Affymetrix\n",
      "!Platform_manufacture_protocol = The sequence information for this array was selected from public data sources GenBankÂ® (May 2004) and Sanger Center (June 2004) for the S. cerevisiae and S. pombe genomes, respectively. Probe sets on the array include 11 oligonucleotide pairs to detect each transcript.\n",
      "!Platform_catalog_number = 900553\n",
      "!Platform_support = silicon\n",
      "!Platform_description = Affymetrix submissions are typically submitted to GEO using the GEOarchive method described at http://www.ncbi.nlm.nih.gov/projects/geo/info/geo_affy.html\n",
      "!Platform_description =\n",
      "!Platform_description = The GeneChipÂ® Yeast Genome 2.0 Array contains probe sets to detect transcripts from both Saccharomyces cerevisiae and Schizosaccharomyces pombe, which are the two most commonly studied species of yeast. Providing comprehensive coverage of both species, the GeneChip Yeast Genome 2.0 Array includes approximately 5,744 probe sets for 5,841 of the 5,845 genes present in S. cerevisiae and 5,021 probe sets for all 5,031 genes present in S. pombe\n",
      "!Platform_web_link = http://www.affymetrix.com/support/technical/byproduct.affx?product=yeast-20\n",
      "!Platform_web_link = http://www.affymetrix.com/analysis/index.affx\n",
      "!Platform_contact_name =  ,,Affymetrix, Inc.\n",
      "!Platform_contact_email = geo@ncbi.nlm.nih.gov, support@affymetrix.com\n",
      "!Platform_contact_phone = 888-362-2447\n",
      "!Platform_contact_institute = Affymetrix, Inc.\n",
      "!Platform_contact_address =\n",
      "!Platform_contact_city = Santa Clara\n",
      "!Platform_contact_state = CA\n",
      "!Platform_contact_zip/postal_code = 95051\n",
      "!Platform_contact_country = USA\n",
      "!Platform_contact_web_link = http://www.affymetrix.com/index.affx\n",
      "!Platform_relation = Alternative to: GPL9671 (Alternative CDF)\n",
      "!Platform_relation = Alternative to: GPL13150 (Alternative CDF)\n",
      "!Platform_data_row_count = 10928\n",
      "!platform_table_begin\n",
      "!platform_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient0h_rep1\n",
      "!Sample_geo_accession = GSM564449\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 0h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0 hours sample (steady state) from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564449/suppl/GSM564449_1_Ys30_7.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient0.2h_rep1\n",
      "!Sample_geo_accession = GSM564450\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 0.2h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0.2h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0.2 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564450/suppl/GSM564450_2_Ys30_8.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient1h_rep1\n",
      "!Sample_geo_accession = GSM564451\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 1h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 1h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 1 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564451/suppl/GSM564451_3_Ys30_9.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient3h_rep1\n",
      "!Sample_geo_accession = GSM564452\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 3h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 3h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 3 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564452/suppl/GSM564452_4_Ys30_11.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient8h_rep1\n",
      "!Sample_geo_accession = GSM564453\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 8h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 8h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 8 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564453/suppl/GSM564453_5_Ys30_16.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient24h_rep1\n",
      "!Sample_geo_accession = GSM564454\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 24h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 24h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 24 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564454/suppl/GSM564454_6_Ys30_17.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient72hh_rep1\n",
      "!Sample_geo_accession = GSM564455\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 72hh.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 72h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 72 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564455/suppl/GSM564455_7_Ys30_20.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient0h_rep2\n",
      "!Sample_geo_accession = GSM564456\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 0h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0 hours sample (steady state) from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564456/suppl/GSM564456_8_Ys31_7.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient0.2h_rep2\n",
      "!Sample_geo_accession = GSM564457\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 0.2h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0.2h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0.2 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564457/suppl/GSM564457_9_Ys31_8.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient1h_rep2\n",
      "!Sample_geo_accession = GSM564458\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 1h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 1h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 1 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564458/suppl/GSM564458_10_Ys31_9.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient3h_rep2\n",
      "!Sample_geo_accession = GSM564459\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 3h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 3h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 3 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564459/suppl/GSM564459_11_Ys31_11.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient8h_rep2\n",
      "!Sample_geo_accession = GSM564460\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 8h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 8h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 8 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564460/suppl/GSM564460_12_Ys31_16.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient24h_rep2\n",
      "!Sample_geo_accession = GSM564461\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 24h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 24h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 24 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564461/suppl/GSM564461_13_Ys31_17.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_1%oxygen_transient72h_rep2\n",
      "!Sample_geo_accession = GSM564462\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 1% to 0% oxygen, timepoint 72hh.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 1% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 72h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 72 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 1 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564462/suppl/GSM564462_14_Ys31_20.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient0h_rep1\n",
      "!Sample_geo_accession = GSM564463\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 0h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0 hours sample (steady state) from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564463/suppl/GSM564463_15_Ys32_8.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient0.2h_rep1\n",
      "!Sample_geo_accession = GSM564464\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 0.2h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0.2h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0.2 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564464/suppl/GSM564464_16_Ys32_9.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient1h_rep1\n",
      "!Sample_geo_accession = GSM564465\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 1h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 1h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 1 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564465/suppl/GSM564465_17_Ys32_10.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient3h_rep1\n",
      "!Sample_geo_accession = GSM564466\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 3h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 3h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 3 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564466/suppl/GSM564466_18_Ys32_12.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient8h_rep1\n",
      "!Sample_geo_accession = GSM564467\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 8h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 8h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 8 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564467/suppl/GSM564467_19_Ys32_17.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient24h_rep1\n",
      "!Sample_geo_accession = GSM564468\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 24h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 24h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 24 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564468/suppl/GSM564468_20_Ys32_18.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient79h_rep1\n",
      "!Sample_geo_accession = GSM564469\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 79h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 79h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 79 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564469/suppl/GSM564469_21_Ys32_21.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient0h_rep2\n",
      "!Sample_geo_accession = GSM564470\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 0h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0 hours sample (steady state) from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564470/suppl/GSM564470_22_Ys33_8.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient0.2h_rep2\n",
      "!Sample_geo_accession = GSM564471\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 0.2h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 0.2h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 0.2 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564471/suppl/GSM564471_23_Ys33_9.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient1h_rep2\n",
      "!Sample_geo_accession = GSM564472\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 1h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 1h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 1 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564472/suppl/GSM564472_24_Ys33_10.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient3h_rep2\n",
      "!Sample_geo_accession = GSM564473\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 3h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 3h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 3 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564473/suppl/GSM564473_25_Ys33_12.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient8h_rep2\n",
      "!Sample_geo_accession = GSM564474\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 8h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 8h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 8 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564474/suppl/GSM564474_26_Ys33_17.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient24h_rep2\n",
      "!Sample_geo_accession = GSM564475\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 24h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 24h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 24 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564475/suppl/GSM564475_27_Ys33_18.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n",
      "SAMPLE \n",
      "!Sample_title = Yeast_20.9%oxygen_transient79h_rep2\n",
      "!Sample_geo_accession = GSM564476\n",
      "!Sample_status = Public on Apr 01 2011\n",
      "!Sample_submission_date = Jul 08 2010\n",
      "!Sample_last_update_date = Apr 01 2011\n",
      "!Sample_type = RNA\n",
      "!Sample_channel_count = 1\n",
      "!Sample_source_name_ch1 = Saccharomyces cerevisiae cells, glucose-limited chemostat, transient from 20.9% to 0% oxygen, timepoint 79h.\n",
      "!Sample_organism_ch1 = Saccharomyces cerevisiae\n",
      "!Sample_taxid_ch1 = 4932\n",
      "!Sample_characteristics_ch1 = protocol: transition from 20.9% to 0% oxygen\n",
      "!Sample_characteristics_ch1 = time: 79h\n",
      "!Sample_treatment_protocol_ch1 = none\n",
      "!Sample_growth_protocol_ch1 = Cells were grown in 0.8â€“1 L of medium in B. Braun Biotech International (Sartorius AG, Germany) Biostat CT (2.5 Lworking volume) bioreactors. Cultures were inoculated to aninitial OD600 nm of c. 0.5, and maintained as batch culturesfor 6â€“9 h, after which continuous medium feed was started while the cells were still growing exponentially. Chemostatcultures were maintained at D= 0.10 /h, pH 5.0, and30C, with 1.5 volume gas [volume culture]/min (vvm).For cultures that received 20.9% O2 in the gas stream, O2was replaced with the equivalent volume of N2, so that totalgas flow was kept constant for all experiments. Steady-state samples were taken after the cultures had beenin constant conditions for a minimum of four residence times (six generations). Steady states were assessed over fourto nine residence times (six to 13 generations) for constantbiomass production, carbon dioxide evolution and oxygenuptake rates (CER and OUR), alkali utilization, and extracellularmetabolites, as well as constant intracellular metabolites and gene transcription. Cultures that were fed 20.9% O2 were subject tooscillations. To prevent these, c. 5% of the total cellconcentration in the bioreactor was added to the culture ascells in mid-exponential to late exponential phase at thetime when continuous medium feed was started (Zamamiri et al., 2001).\n",
      "!Sample_molecule_ch1 = total RNA\n",
      "!Sample_extract_protocol_ch1 = 10 ml of cell culture was centrifuged for 5 min, 3500 rpm, +4C, cells were frozen in liquid nitrogen and stored at -80C. Total RNA was extracted using QIAGEN RNeasy Mini Kit (QIAGEN, Inc., Valencia, CA) according to the manufacturers instructions with the following modification to the cell lysis step. 5-20 mg dry mass of cells were suspended in 400Âµl of cold (+4Â°C) disruption buffer (20mM Tris-HCl, pH 7.4, 100 mM KCl, 2 mM MgCl2,  2 mM DTT). 400Âµl of phenol-chlorophorm (50:50), 5Âµl 20% SDS and 400 Âµl glass beads (0.5 mm diameter; Biospec Products) was added. The cells were disrupted with Fastprep machine (Q-Biogene), 2 x 20s, at speed 6. After centrifugation (14000 rpm, 15 min, +4Â°C), supernatant was used for total RNA extraction. RNA quality was checked with Agilent bioanalyzer (Agilent Technologies).\n",
      "!Sample_label_ch1 = biotin\n",
      "!Sample_label_protocol_ch1 = Approximately 2 Âµg of total RNA was processed to produce biotinylated cRNA targets using Affymetrix One-Cycle Target prep protocol.\n",
      "!Sample_hyb_protocol = Hybridisation was done at +45Â°C overnight (16Â h) according to GeneChip Expression Analysis Technical Manual (Affymetrix). GeneChip Fluidics Station 450 was used to wash and stain the arrays.\n",
      "!Sample_hyb_protocol = GeneChip Scanner 3000 with AutoLoader was used to scan the arrays.\n",
      "!Sample_scan_protocol = GeneChip Scanner 3000 with AutoLoader\n",
      "!Sample_description = Study description: S. cerevisiae CEN.PK113-1A (MATa, URA3, HIS3, LEU2, TRP1, MAL2-8c, SUC2) grown in Verduyn mimimal medium with 10 g/L glucose as carbon source, and supplemented with 10 mg/L ergosterol and 420 mg/L Tween 80, in Biostat CT  bioreactor. Sample from steady state with 1% oxygen, transient timepoint 0h\n",
      "!Sample_description = This sample: This sample: 79 hours sample from a time-series experiment. Glucose-limited chemostat, starting oxygen provision 20.9 %. After steady state was achieved, oxygen feed was switched to nitrogen for anaerobic conditions.\n",
      "!Sample_data_processing = RMA normalised using R/Bioconductor (rma package), version 2.5.1.\n",
      "!Sample_data_processing = The control probes and S.pombe probes were removed from the data before RMA normalisation. Thus the RMA processed data, reported in the Sample table, does not include control and S.pombe probes. The raw data of the removed probes is present in the raw data (.CEL) files.\n",
      "!Sample_platform_id = GPL2529\n",
      "!Sample_contact_name = Eija,,Rintala\n",
      "!Sample_contact_email = eija.rintala@vtt.fi\n",
      "!Sample_contact_institute = VTT Technical Research Centre of Finland\n",
      "!Sample_contact_address = P.O.BOX 1000\n",
      "!Sample_contact_city = VTT\n",
      "!Sample_contact_zip/postal_code = 02044\n",
      "!Sample_contact_country = Finland\n",
      "!Sample_supplementary_file = ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM564nnn/GSM564476/suppl/GSM564476_28_Ys33_21.CEL.gz\n",
      "!Sample_series_id = GSE22832\n",
      "!Sample_data_row_count = 5716\n",
      "!sample_table_begin\n",
      "!sample_table_end\n"
     ]
    }
   ],
   "source": [
    "\n",
    "def read_soft_file(file_path):\n",
    "    data = []\n",
    "    \n",
    "    with gzip.open(file_path, 'rt') as f:\n",
    "        current_section = None\n",
    "        current_data = {}\n",
    "\n",
    "        for line in f:\n",
    "            line = line.strip()\n",
    "            \n",
    "            if line.startswith('^'):\n",
    "                current_section = line.split('=')[0][1:]\n",
    "                print(current_section)\n",
    "            elif line.startswith('!'):\n",
    "                print(line)\n",
    "\n",
    "\n",
    "filepath = 'GSE22832_family.soft.gz'\n",
    "\n",
    "read_soft_file(filepath)\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "if line.startswith(\"^\"):\n",
    "                current_section = line[1:]\n",
    "                if current_data:\n",
    "                    data.append(current_data)\n",
    "                    current_data = {}\n",
    "            elif current_section:\n",
    "                key, value = line.split(\" = \", 1)\n",
    "                current_data[key] = value\n",
    "\n",
    "    if current_data:\n",
    "        data.append(current_data)\n",
    "\n",
    "    return data\n",
    "\\\n",
    "\n",
    "# Read the file and store the data in a data structure\n",
    "soft_data = read_soft_file(file_path)\n",
    "\n",
    "# Convert the data to a Pandas DataFrame for easier analysis\n",
    "df = pd.DataFrame(soft_data)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
